sample-app/src/components/VisualAutocomplete/SampleVisualSearchBar.tsx

```diff
@@ -0,0 +1,78 @@
+import VisualSearchBar from './VisualSearchBar';
+import { Result } from '@yext/answers-headless-react';
+import VisualEntities from './VisualEntities';
+
+/**
+ * This is an example of how to use the VisualSearchBar component.
+ */
+export default function SampleVisualSearchBar() {
+  return (
+    <VisualSearchBar
+      placeholder='Search...'
+      screenReaderInstructionsId='SearchBar__srInstructions'
+      headlessId='visual-autocomplete'
+      debounceTime={150}
+      renderVisualEntities={(isLoading) => (
+        <div className={isLoading ? 'opacity-50' : ''}>
+          <VisualEntities verticalKey='people'>
+            {results => (
+              <div className='flex'>
+                {results.map(r => <PeopleCard result={r} key={r.name} />)}
+              </div>
+            )}
+          </VisualEntities>
+          <VisualEntities verticalKey='faq' limit={2}>
+            {results => (
+              <div className='flex flex-col'>
+                {results.map(r => <FaqCard result={r} key={r.name} />)}
+              </div>
+            )}
+          </VisualEntities>
+        </div>
+      )}
+    >
+      {(autocompleteDropdown, visualEntities) => 
```

Comment:
Have you seen this pattern elsewhere, where you pass the layout of the Component as `props.children`? I can't remember if it's in the Bootstrap example we talked about. If the spec is unclear, we should as Product how important it is for v1 of VA to have a configurable layout like this. If the answer is "people have to fork the Component" to change the layout, it would make our lives easier.

Reply:
afaik FACC is typically used for exposing some data to the user, not JSX nodes. It is not included in the react-bootstrap, which seems to exclusively use the compound component pattern instead. Both do a similar thing here - which is to provide some set functionality without prescribing much UI. I think compound component here would be cleaner and also more flexible, but we would probably need to rework DropdownSection.